client/scss/components/forms/_field-comment-control.scss

```diff
@@ -47,15 +44,17 @@ $button-size: theme('spacing.5');
   }
 }
 
-// Add extra space to the right of commentable fields for the buttons.
 .w-field--commentable {
+  // Allow comment buttons to be placed along side the field by display flex
   .tab-content--comments-enabled & .w-field__input {
-    padding-inline-end: $button-size;
+    display: flex;
```

Comment:
I find the switch from absolute positioning to flexbox problematic in this instance. We generally shouldn’t assume anything about the rendering of fields’ widgets. Making this `w-field__input` use Flexbox means that widgets with more than one top-level element in their template won’t render correctly.

We’d either need to be ok with that, or add another wrapper around widgets to side-step the issue, or go back to absolute positioning.

Regardless of which option we go for – now that Wagtail 4.0 is out, I think we should try an approach where we update the fields’ rendering regardless of whether comments are enabled or not, and regardless of whether fields support commenting or not. This will prevent commenting-specific regressions that would be hard to spot otherwise.